spec/system/decidim/action_delegator/admin/admin_manages_settings_spec.rb

```diff
@@ -41,6 +41,59 @@
     end
   end
 
+  context "when creating with copy from other setting" do
+    let!(:first_consultation) { create(:consultation, organization: organization) }
+    let!(:setting) { create(:setting, :with_participants, :with_ponderations, authorization_method: :both, consultation: first_consultation) }
+
+    before do
+      switch_to_host(organization.host)
+      login_as user, scope: :user
+      visit decidim_admin.users_path
+      click_link I18n.t("decidim.action_delegator.admin.menu.delegations")
+
+      click_link I18n.t("decidim.action_delegator.admin.settings.index.actions.new_setting")
+    end
+
+    it "creates a new setting" do
+      fill_in :setting_max_grants, with: 5
+      select translated_attribute(consultation.title), from: :setting_decidim_consultation_id
+      select translated_attribute(first_consultation.title), from: :setting_copy_from_setting_id
+
+      find("*[type=submit]").click
+
+      expect(page).to have_admin_callout("successfully")
+      expect(page).to have_content(translated_attribute(first_consultation.title))
+      expect(page).to have_css("tr[data-setting-id='#{setting.id}'] td:nth-of-type(5)", text: "3")
+      expect(page).to have_css("tr[data-setting-id='#{setting.id}'] td:nth-of-type(6)", text: "3")
```

Comment:
I don't understand this test, shouldn't be we checking the newly created settings if they have the same participants & ponderations as the old (but with new ids)